Zend/tests/type_declarations/union_types/null_false_union.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Null and false cann be used in a union type
+--FILE--
+<?php
+
+function test1(): null|false {}
```

Comment:
nit: Would be useful and help readability to call these and test the expected behavior, and have example implementations where `return null;` is properly included in the implementation.

Readers of the RFC might be looking at the tests to see how it's used

```
Warning: Uncaught TypeError: Return value of test1() must be of the type false or null, none returned....
```

Reply:
I'll add such a test case